README.md

```diff
@@ -9,4 +9,4 @@ This repo will contain if statements for every coded this project started on 04/
 
 ## Creadits
 
-Silverkill (No public github - will link when I get the opatunity since hes the whole reason this joke exists)
```

Comment:
Think there might be an issue here. Have linked github for the user the repo is on?

Reply:
No issues all intentional